packages/space-opera/src/components/camera_settings/components/limits_base.ts

```diff
@@ -53,9 +53,9 @@ export abstract class LimitsBase extends ConnectedLitElement {
     return 1;
   }
 
-  @query('#minimum') minimumInput?: SliderWithInputElement;
-  @query('#maximum') maximumInput?: SliderWithInputElement;
-  @query('#limit-enabled') enabledInput?: HTMLInputElement;
+  @query('#minimum') minimumInput!: SliderWithInputElement;
+  @query('#maximum') maximumInput!: SliderWithInputElement;
+  @query('#limit-enabled') enabledInput!: HTMLInputElement;
```

Comment:
What was the reasoning behind the original design?
It looks like previously it was ok for any of these variables to be undefined, and now we are saying it shouldn't?

Reply:
Yeah, I've been slowly updating this style across the editor. Before when  a panel  was disabled, they replaced the html wholesale  with nothing, while now I'm preferring to set `display: 'none'`. In this way the elements always exist  and a lot of checking can be removed. In some cases, they had `?` even if  they already always existed, probably because  this is the TS default.